### PR TITLE
motion data: two example datasets (BEP029)

### PR DESCRIPTION
**Pull request by sgrothk**
to provide example motion datasets as proposed in BEP029 for motion data:
- dualsystem: multiple task per session with simultaneous IMU and optical motion capture example
- dualtask: multisession with single optical motion capture system and simultaneous eeg recording example
- spotrotation: multisession with virtual and physical system and simultaneous eeg recording example

See [Google Docs](https://docs.google.com/document/d/1iaaLKgWjK5pcISD1MVxHKexB3PZWfE2aAC5HF_pCZWo/edit#heading=h.w5vg69wsp4vc) for BEP029

### CODE REVIEW

**Comment by sappelhoff** [comment]
two notes:

1. I changed the target of this PR from `master` to a new branch [`bep029_motion`](https://github.com/bids-standard/bids-examples/tree/bep029_motion)
2. it would be best to submit each example dataset with a separate PR, to get a better overview

Now that we have a dedicated branch for BEP029, we might as well merge this PR and perform future improvements in follow up PRs. Finally, when BEP029 is completed, we will merge the `bep029_motion` branch into `master`.

**Comment by sappelhoff** [comment]
in case you wonder about me changing the base of the branch, that's because you merged "master" into your branch -- but the actual target of this PR is "bep029_motion" (you can see that at the top of the PR, below the title). 

The changes you merged from "master" were not yet in "bep029_motion", so GitHub wanted to add these changes as well, which pollutes the commit history and diff of the PR. 

I solved this by updating "bep029_motion" (via `git rebase master` or `git merge master` and then pushing) ... but for this to be reflected in the PR, you need to change the base to another branch and then back (a bit hacky, but apparently that's needed for some GitHub scripts to get triggered).

Same in https://github.com/bids-standard/bids-examples/pull/329

TL;DR: everything is fine

**Comment by sappelhoff** [comment]
see https://github.com/bids-standard/bids-examples/pull/335#issue-1360822464 for reasons why I closed this.
